<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creature</class>
 <widget class="QFrame" name="Creature">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="m_element">
       <property name="text">
        <string>Element</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_academicMagic">
       <property name="text">
        <string>Academic Magic</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_role">
       <property name="text">
        <string>Role</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="m_icon">
         <property name="text">
          <string>CreatureIcon</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="m_name">
           <property name="text">
            <string>CreatureName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_id">
           <property name="text">
            <string>CreatureId</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_level">
           <property name="text">
            <string>CreatureLevel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="m_families">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="m_runes">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="m_stats">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
